packages/template/README.md

```diff
@@ -2,84 +2,36 @@
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
-**Template** is a [brief description of your library's purpose].
+**Template** is a package that allows you to generate new packages with ease. It provides a convenient command-line interface for generating packages based on predefined templates.
 
 ## Features
 
-- [List key features or functionalities of your library.]
+- Easily generate new packages with a single command
+- Customizable templates for different project types
+- Supports popular configurations like Vite, Vitest, and TypeScript
 
-## Installation
+## Installation and usage
 
-add template to dependencies in package.json
+To use the **Template** package as a template to generate your next package, you don't need to install it inside your monorepo. Simply follow the instructions below:
 
-```json
+1. Make sure you have Node.js installed on your machine.
 
-  "dependencies": {
-    "@cloudoperators/juno-template": "*"
-  },
+2. Open your terminal and navigate to the root directory of your project.
 
-```
-
-## Usage
-
-Here's how you can use **template** in your project:
-
-```javascript
-// Example code or usage instructions
-```
-
-## Testing
-
-To run tests, use the following command:
-
-```bash
-npm run test
-```
-
-If you're working within the Juno monorepo using workspaces, you can use:
-
-```bash
-npm -w template run test
-```
-
-Alternatively, from within the workspace:
+3. Run the following command to install the **Template** package as a global dependency:
 
 ```bash
-wb npm -w template run test
+turbo gen workspace --type package --copy @cloudoperators/juno-package-template
```

Comment:
don't you want to reference here to the task `generate:package`?

Reply:
This is on package lvl is more correct, that task that you suggested is more on root lvl